jina/flow/base.py

```diff
@@ -605,6 +616,83 @@ def gather_inspect(
             # no inspect node is in the graph, return the current graph
             return self
 
+    def _get_gateway_target(self, prefix):
+        gateway_pod = self._pod_nodes[GATEWAY_NAME]
+        return (
+            f'{prefix}-{GATEWAY_NAME}',
+            {
+                'host': gateway_pod.head_host,
+                'port': gateway_pod.head_port_in,
+                'expected_parts': 0,
+            },
+        )
+
+    def _update_port_type(self, pod):
+        pod.head_args.socket_in = SocketType.ROUTER_BIND
+        pod.tail_args.dynamic_out_routing = True
+        pod.tail_args.socket_out = SocketType.DEALER_CONNECT
+
+    def _get_routing_graph(self):
+        target_pods = [
+            (
+                pod_id,
+                {
+                    'host': pod.head_host,
+                    'port': pod.head_port_in,
+                    'expected_parts': 0,
+                },
+            )
+            for pod_id, pod in self._pod_nodes.items()
+            if pod_id != GATEWAY_NAME
+        ]
+        target_pods = (
```

Comment:
why do not unify in the line above?

Reply:
I changed the naming of the above variable and it should be better now.